README.md

```diff
@@ -1,65 +1,25 @@
 [![Join the chat at https://gitter.im/openshift/openshift-ansible](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/openshift/openshift-ansible)
 [![Build Status](https://travis-ci.org/openshift/openshift-ansible.svg?branch=master)](https://travis-ci.org/openshift/openshift-ansible)
-[![Coverage Status](https://coveralls.io/repos/github/openshift/openshift-ansible/badge.svg?branch=master)](https://coveralls.io/github/openshift/openshift-ansible?branch=master)
-
-NOTICE
-======
-
-Master branch is closed! A major refactor is ongoing in devel-40.
-Changes for 3.x should be made directly to the latest release branch they're
-relevant to and backported from there.
-
-WARNING
-=======
-
-This branch is under heavy development.  If you are interested in deploying a
-working cluster, please utilize a release branch.
 
 # OpenShift Ansible
-
 This repository contains [Ansible](https://www.ansible.com/) roles and
-playbooks to install, upgrade, and manage
-[OpenShift](https://www.openshift.com/) clusters.
-
-## Getting the correct version
-When choosing an openshift release, ensure that the necessary origin packages
-are available in your distribution's repository.  By default, openshift-ansible
-will not configure extra repositories for testing or staging packages for
-end users.
-
-We recommend using a release branch. We maintain stable branches
-corresponding to upstream Origin releases, e.g.: we guarantee an
-openshift-ansible 3.2 release will fully support an origin
-[1.2 release](https://github.com/openshift/openshift-ansible/tree/release-1.2).
-
-The most recent branch will often receive minor feature backports and
-fixes. Older branches will receive only critical fixes.
+playbooks for [OpenShift](https://www.openshift.com/) clusters.
 
-In addition to the release branches, the master branch
-[master branch](https://github.com/openshift/openshift-ansible/tree/master)
-tracks our current work **in development** and should be compatible
-with the
-[Origin master branch](https://github.com/openshift/origin/tree/master)
-(code in development).
+## Previous OpenShift Ansible 3.x releases
+For 3.x releases of OpenShift Ansible please reference the release branch for
+specific versions.  The last 3.x release is 
+[3.11 release](https://github.com/openshift/openshift-ansible/tree/release-3.11).
 
+## OpenShift 4
+Installation of OpenShift beginning with 4.0 uses a command-line installation
+wizard instead of Ansible playbooks.  Learn more about the OpenShift Installer
+in this [overview](https://github.com/openshift/installer/blob/master/docs/user/overview.md#installer-overview).
 
+For OpenShift 4.0, this repo only provides playbooks necessary for scaling up an
+existing 4.0 cluster with RHEL hosts.
```

Comment:
is this the case for 4.0 only while on 4.1+ the scope will be extended ?

regarding _RHEL hosts_ are are pinned to a particular version - ie >= 8.0 ?

Reply:
Thanks for the input.  In this paragraph, I was attempting to provide a contrast between 3.x and 4.0 in that they are very different, but not get bogged down in a specific feature inclusion/exclusion list.  Something like, "Don't come looking for installation playbooks, all you'll find right now is a scaleup playbook."  I was also trying to avoid '4.x'.  I've pushed a small update.

Reply:
fair point, thx for the info